### ISSUE TITLE
Create Fall Back Twitter Service

### ISSUE THREAD

**Issue by pegwymonie** [brutesquaddev/twitterific#5]
Flesh out the mock twitter service to contain enough information to stand in for the real twitter service. We will use this to fall back to when the main Twitter service is not available.

**Comment by tealamore** [comment]
Good idea @SoloTSi97. We should set up a meeting to flesh out the requirements of this issue.

**Comment by SoloTSi97** [comment]
For sure. I'd look to @pegwymonie to set something up.

**Comment by tealamore** [comment]
@frysinbr Do you have any thoughts about this?

**Comment by bradfrys** [comment]
Looks like an intern wrote this.

**Comment by tealamore** [comment]
@pegwymonie @bradfrys @SoloTSi97 @dentinger  yo what's up
